### ISSUE TITLE
Call direct intersegment not handled?

### ISSUE THREAD

**Issue by jgbpercy** [cmuratori/computer_enhance#3]
So I could totally be misunderstanding, but I don't think the reference decoder is handling "Call direct intersegment"

If I give NASM this assembly
```
bits 16 ; or cpu 8086 - same result
call 999:888
```
I get these bytes (hex)
```
9A 78 03 E7 03
```
Giving this to the reference decoder I get this output
```
bits 16
call 231
ERROR: Instruction extends outside disassembly region
```
Unless this isn't an 8086 instruction? But those by bytes do look to me like that instruction.

(Built on Windows 10 MSVC. Other listing work as expected. Believe the same also applies to "Jump direct intersegment")

**Comment by cmuratori** [comment]
THANK YOU! I had no idea what the syntax in NASM was for creating intersegment call (and jmp has this too), so it wasn't in the test files. It looks like I set up the table wrong - it should have a DATA_IF_W as well as DATA, and it doesn't, so it's only reading 8 bits for the second parameter instead of 16. That's why you're getting the segment boundary error.

Thanks to your test case, I have fixed the table and added the ability to print, it will be in the next update (and I will update the completionist homework to include this syntax).

\- Casey
